betanets/interop-rc-betanet/interop-rc-beta-1/chain.yaml

```diff
@@ -0,0 +1,46 @@
+name: interop-rc-beta-1
+chain_id: 420110010
+rpc_url: https://interop-rc-beta-1.optimism.io
+contracts:
+  superchainDeployment:
+    proxyAdminAddress: 0x26DF14a0C889de2448d228Ee23B2530550b5B774
+    superchainConfigProxyAddress: 0x584ceae10f7d1b485b01b9df49557c6b8721bd11
+    superchainConfigImplAddress: 0x4eee58d4c012713b23b6cb73ba64236eff52eb6e
+    protocolVersionsProxyAddress: 0x50d1f0541b03c89a149e0e59256e8871665176cb
+    protocolVersionsImplAddress: 0x37e15e4d6dffa9e5e320ee1ec036922e563cb76c
+  opChainDeployment:
+    proxyAdminAddress: 0x19397254e9387dee2cf60142989428706348c265
+    addressManagerAddress: 0x71bf7b69110d20b0ab728aacae6818463f258278 
+    l1ERC721BridgeProxyAddress: 0x14929b7291cfcbf7b8446de291d5ca8ed78e7cd6
+    systemConfigProxyAddress: 0xc7ba6cba614bc17540e7b2cd9844d45efcaa0609
+    optimismMintableERC20FactoryProxyAddress: 0xc2838817730258305e43c904aad3a6d97cdfd397
+    l1StandardBridgeProxyAddress: 0x1cafac6b17abe0386068d1ab4c32520b74ff97de
+    l1CrossDomainMessengerProxyAddress: 0x8832bfb936857d84c51331707e9d70a11cb29955
+    optimismPortalProxyAddress: 0xa56700196e5e0564733d7b9f0c4325c868215685
+    ethLockboxProxyAddress: 0x02a38b90c4dbf2cbfd0d7899421f7423e0622d7d
+    disputeGameFactoryProxyAddress: 0xaff9e60c22c21f6849ab73b55755bf7ab550e263
+    anchorStateRegistryProxyAddress: 0xe91b4069b8cbab7dc621c3525b462853467ccf02
+    faultDisputeGameAddress: 0x0000000000000000000000000000000000000000
+    permissionedDisputeGameAddress: 0x45bbee9272d8e4a592c70f7ef4f00cdd7c3332da
+    delayedWETHPermissionedGameProxyAddress: 0x4d353678850e1ad53b965974489e7a4fa184eec1
+    delayedWETHPermissionlessGameProxyAddress: 0x0000000000000000000000000000000000000000
```

Comment:
In the future, we should ensure the permissionless delayed weth proxy is deployed.